testing/main_test.go

```diff
@@ -147,9 +154,13 @@ func testORM(
 		orm:      orm,
 	}
 
-	db, dbURL, stopDB := initTestDatabase(t, app)
+	db, dbURL, version, stopDB := initTestDatabase(t, app)
 	defer stopDB()
 
+	if orm == "django"  && (strings.HasPrefix(version, "v2.0") || strings.HasPrefix(version, "v2.1")) {
+		t.Skip("TestDjango fails on CRDB <v2.1 due to missing foreign key support.")
```

Comment:
Looks like you're skipping 2.1 here.

Reply:
yeah, we need to skip 2.0 and 2.1 since Django won't work with them

Reply:
I see, just update the comment to "<=v2.1" then.

Reply:
ah, good call